alarm/dkms-8812au/dkms.conf

```diff
@@ -0,0 +1,11 @@
+PACKAGE_NAME="@PKGBASE@"
+PACKAGE_VERSION="@PKGVER@"
+BUILT_MODULE_NAME="8812au"
+PROCS_NUM=$(nproc)
+[ $PROCS_NUM -gt 16 ] && PROCS_NUM=16
+MAKE="'make' ARCH=arm64 -j$PROCS_NUM KVER=${kernelver} 
```

Comment:
This package is for all architectures, but the build line has ARCH=arm64?

Reply:
Nice catch and sorry for that, I totally forgot about that work around for the wrong detection code on the Makefile, you really have super eyes! Fixed it by implementing some basic replacement logic on the PKGBUILD for aarch64 and arm64 that should always be arm64 if one of both is detected.